src/qmapshack/dem/CDemVRT.cpp

```diff
@@ -325,6 +326,12 @@ void CDemVRT::draw(IDrawContext::buffer_t& buf)
     qreal nTiles = ((right - left) * (bottom - top) / (w * h));
     if(!isOutOfScale(bufferScale) && (nTiles < TILELIMIT))
     {
+	// get conversion factor between meters and elevation units
+	//
+	qreal elevationConversion;
```

Comment:
IMHO `elevationFactor` would be the better naming. 

Or better: Have a look at `IUnit` and `speedfactor` for an example. This is a much more lightweight way to access these factors.

Reply:
conversion of elevation units can also been done in IDem::elevationLimit().
I'll commit a change for review.